File: facedetection-java/src/main/java/com/facecool/attendance/facedetector/ModelConfig.java
package com.facecool.attendance.facedetector;

import androidx.annotation.Keep;

@Keep
public class ModelConfig {
    float scale = 0.0F;
    float shift_x = 0.0F;
    float shift_y = 0.0F;
    int height = 0;
    int width = 0;
    String name = "";
    boolean org_resize = false;
}

File: facedetection/src/main/java/com/facecool/attendance/facedetector/ModelConfig.kt
package com.facecool.attendance.facedetector

import androidx.annotation.Keep

@Keep
internal class ModelConfig {
    var scale = 0.0f
    var shift_x = 0.0f
    var shift_y = 0.0f
    var height = 0
    var width = 0
    var name = ""
    var org_resize = false
}